pkgs/applications/editors/neovim/wrapper.nix

```diff
@@ -11,33 +11,44 @@ neovim:
 
 let
   wrapper = {
-      # should contain all args but the binary
-      wrapperArgs ? ""
+      extraName ? ""
+    # should contain all args but the binary. Can be either a string or list
+    , wrapperArgs ? []
     , manifestRc ? null
     , withPython2 ? false
     , withPython3 ? true,  python3Env ? null
     , withNodeJs ? false
     , rubyEnv ? null
     , vimAlias ? false
     , viAlias ? false
+
+    # additional argument not generated by makeNeovimConfig
+    # it will append "-u <customRc>" to the wrapped arguments
+    # set to false if you want to control where to save the generated config
+    # (e.g., in ~/.config/init.vim or project/.nvimrc)
+    , wrapRc ? true
```

Comment:
Is seems that this change now adds a `-u` flag where there wasn't before causing this PR to prevent the loading of user RC files by default. Should this be flipped to `false` to match the previous default quickly?

It seems very surprising that the program doesn't read user RC files by default, perhaps this would be better suited in the NixOS module instead of the program itself?

Reply:
in latest master, in neovim/utils.nix its value is set to `wrapRc = (configure != {});`.
What rev are you using and wha'ts your config ?

Reply:
I'm using the latest NixOS channel `.nix-defexpr/channels/nixos -> /nix/store/1sng5sppcarfw57pw49pn1d8d2slnvaz-nixos-21.11pre292748.6933d068c5d/nixos`.

My config is:

```nix
        environment.systemPackages = with pkgs; [
                neovim         
        ];
```

Reply:
Ok, so it appears that the fix is 6933d068c5d which hasn't been released yet. I'm a little confused because https://status.nixos.org/ claims that nixos-unstable is only 6h old but I guess that is a different issue because 6933d068c5d is definitely more than 6h old.

Reply:
agreed that it shouldn't wrap it, I guess it wraps it with an empty config ? either we could default to false (kinda like previously) or wrap (by default) only when needed (e.g., plugins configured, some customRc).

Reply:
Test suite sounds good. I guess I'll just rollback for now and wait until the channel actually updates 😅